src/engine/input/key-mapping/InputGroupItem.cs

```diff
@@ -0,0 +1,40 @@
+﻿using System.Collections.Generic;
+using Godot;
+using Newtonsoft.Json;
+
+public class InputGroupItem : VBoxContainer
+{
+    [Export]
+    [JsonIgnore]
+    public NodePath InputGroupHeaderPath;
+
+    [Export]
+    [JsonIgnore]
+    public NodePath InputActionsContainerPath;
+
+    private Label inputGroupHeader;
+    private VBoxContainer inputActionsContainer;
+
+    [JsonProperty]
+    public string GroupName { get; set; }
+
+    [JsonProperty]
+    public IList<InputActionItem> Actions { get; set; }
```

Comment:
List should be used rather than IList, unless there's a good reason?

Reply:
There isn't really a reason.
Generally, you should create variables of the interface, not the implementation, but List is a very common class, so you're right.